docs/OV_Runtime_UG/supported_plugins/GPU.md

```diff
@@ -36,122 +47,102 @@ Available devices:
-| `KEY_TUNING_MODE`     | `TUNING_DISABLED` <br /> `TUNING_CREATE` <br />  `TUNING_USE_EXISTING`            | `TUNING_DISABLED` | Disable inference kernel tuning     <br /> Create tuning file (expect much longer runtime)  <br />         Use an existing tuning file. **Deprecated**. Will be removed in the next release |
-| `KEY_TUNING_FILE`     | `"<filename>"`                  | `""`              | Tuning file to create / use. **Deprecated**. Will be removed in the next release |
-
-## Quering GPU specific metric keys
-* MEMORY_STATISTICS : Returns overall memory statistics of `GPU` device allocated by engine with allocation types. If the network has `TensorIterator` or `Loop` operation which is not unrolled, there will be additional allocation at the first inference phase. In such a case, querying for `MEMORY_STATISTICS` should be done after first inference for more accurate result. The code below demonstrates how to query overall memory statistics of `GPU` device:
-
-@snippet snippets/GPU_Metric0.cpp part0
-
-* MAX_BATCH_SIZE : Returns maximum batch size for a given network which is not only executable but also does not lose performance due to the memory swap impact. Note that the returned value may not aligned to power of 2. Also, MODEL_PTR is the required option for this metric since the available max batch size depends on the model size. If the MODEL_PTR is not given, it will return 1. The example code to set the required and optional configs for this metic is available in the following snippet:
-
-@snippet snippets/GPU_Metric1.cpp part1
-
-* OPTIMAL_BATCH_SIZE : Returns _optimal_ batch size for a given network on the given GPU device. The returned value is aligned to power of 2. Also, MODEL_PTR is the required option for this metric since the optimal batch size highly depends on the model. If the MODEL_PTR is not given, the value of 1 is returned. The example code to set the required and optional configs for this metric is available in the following snippet:
-
-@snippet snippets/GPU_Metric1.cpp part2
-## GPU Context and Video Memory Sharing RemoteBlob API
-
-See [RemoteBlob API of GPU Plugin](GPU_RemoteBlob_API.md)
+## Supported inference data types
+GPU plugin supports the following data types as inference precision of internal primitives:
+- f32
+- f16
+- u8
+- i8
+- u1
+
+u1/u8/i8 precision requires model quantization to be done.
+
+Note: Harware acceleration for i8/u8 precision may be unavailable on some platforms. In that case model is executed in floating-point
+precision taken from IR. Hardware support of u8/i8 acceleration can be queried via `ov::device::capabilities` property.
+
+
+## Supported features
+
+### Heterogeneous execution
+GPU plugin supports `ov::Core::query_model()` method, thus it can be used for heterogeneous execution via specifying
+`"HETERO:GPU,CPU"` device instead of `"GPU"`. In that case layers that are not implemented in GPU plugin will be executed via CPU plugin.
```

Comment:
I suppose we can have dedicated "feature sections" only if they describe plugin specific behavior / limitations. If feature just works as expected, we can skip any information except the table on the general plugin's page.. 
How do you think?

Reply:
Hetero section was removed